spark2/src/main/java/org/apache/iceberg/spark/source/Reader.java

```diff
@@ -156,10 +157,15 @@
     this.io = io;
     this.encryptionManager = encryptionManager;
     this.caseSensitive = caseSensitive;
-
-    this.batchReadsEnabled = options.get("vectorization-enabled").map(Boolean::parseBoolean).orElseGet(() ->
-        PropertyUtil.propertyAsBoolean(table.properties(),
-            TableProperties.PARQUET_VECTORIZATION_ENABLED, TableProperties.PARQUET_VECTORIZATION_ENABLED_DEFAULT));
+    Option<String> option = SparkSession.active().sparkContext().getConf()
+            .getOption("spark.iceberg.vectorization.enabled");
+    if (option.isDefined()) {
+      this.batchReadsEnabled = Boolean.valueOf(option.get());
+    } else {
+      this.batchReadsEnabled = options.get("vectorization-enabled").map(Boolean::parseBoolean).orElseGet(() ->
+              PropertyUtil.propertyAsBoolean(table.properties(), TableProperties.PARQUET_VECTORIZATION_ENABLED,
+                      TableProperties.PARQUET_VECTORIZATION_ENABLED_DEFAULT));
```

Comment:
Continuation indents are 4 spaces or 2 indents from the previous. Could you update this?